app/views/application/_navigation.html.slim

```diff
@@ -13,9 +13,9 @@ nav.navbar.navbar-top
       #navbar.collapse.navbar-collapse
         ul.nav.navbar-nav
           - if current_user.superadmin?
-            li= link_to t('users'), users_path
-          li= link_to t('clients'), clients_path
-
+            li= link_to t_title('index', User), users_path
+          - if current_user.staff?
```

Comment:
staff method is ambiguous. One has to look up who is included in the staff and who isn't.